### ISSUE TITLE
Install issue with Godot 3.0.2

### ISSUE THREAD

**Issue by manning390** [nsrosenqvist/godot-wrapper#2]
When using the wrapper to install I got a package error with scons for `xinerama`; scons is used to build Godot. I manually installed the missing package, `libxinerama-dev`, before getting another error in scons with a missing package `xrandr` which is now depreciated. I checked the [dependencies on building Godot 3.0.2 in their docs](http://docs.godotengine.org/en/3.0/development/compiling/compiling_for_x11.html) and found that they were different from what was in the bash script. I swapped them out and it installed/built properly. Since I am only interested in installing the latest stable branch, I am unsure if this breaks past branches/versions.

tl;dr
To properly install 3.0.2-stable (current stable release as of writing) replace line 37 with the following:
```bash
dependencies=(git build-essential scons pkg-config libx11-dev libxcursor-dev libxinerama-dev
    libgl1-mesa-dev libglu-dev libasound2-dev libpulse-dev libfreetype6-dev libssl-dev libudev-dev
    libxi-dev libxrandr-dev) #curl jq
```

**Comment by nsrosenqvist** [comment]
@manning390 Would you mind creating a quick PR with that change and I'll merge it right away? I'm not currently using Godot myself but I reckon other people might appreciate it

**Comment by nsrosenqvist** [comment]
@manning390 nevermind. I got to my computer so I added it myself. Haven't tested the code though but it should work
